operator/pkg/artifacts/registry.go

```diff
@@ -57,7 +57,7 @@ func EnsureRegistrySecretInECNamespace(ctx context.Context, cli client.Client, i
 			}
 		}
 
-		obj.ObjectMeta.Labels = applyECOperatorLabels(obj.ObjectMeta.Labels, "upgrader")
+		obj.ObjectMeta.Labels = applyECOperatorLabels(obj.ObjectMeta.Labels, "upgrader-registry-secret")
```

Comment:
why is this part of this PR?

Reply:
it probably shouldn't be
I was looking through this while trying to figure out the labels for the apply-upgrade job and discovered that we were using the same 'name' label for two different jobs